### ISSUE TITLE
nginx won't accept large save files

### ISSUE THREAD

**Issue by jjcf89** [MajorMJR/factorio-server-manager#14]
@drahti 

Tried uploading save file and recieved the following response from server.  No error was displayed either. Had to inspect network traffic in firefox.

```
<html>
<head><title>413 Request Entity Too Large</title></head>
<body bgcolor="white">
<center><h1>413 Request Entity Too Large</h1></center>
<hr><center>nginx/1.8.1</center>
</body>
</html>
```

**Comment by MajorMJR** [comment]
We should add a line to the Nginx config to allow larger uploads.

client_max_body_size 20m; (or whatever, how large is the savefile? Default for the UI is 20MB)

**Comment by jjcf89** [comment]
My save is ~8M so that works for now.

**Comment by jjcf89** [comment]
Well now i get a different error:
```
<html>
<head><title>500 Internal Server Error</title></head>
<body bgcolor="white">
<center><h1>500 Internal Server Error</h1></center>
<hr><center>nginx/1.8.1</center>
</body>
</html>
```

Nothing unusual in the log.

**Comment by drahti** [comment]
Oh. Because my Upload sucks I didn't acutally test it but copied the save on the server. I'm sorry.
I'll look into this. I'd have expected that `client_max_body_size 20m;` fixes the problem...

**Comment by drahti** [comment]
Ok. This took me quiet a lot longer than expected. I'm not completly sure but I'm guessing that Docker has a permission bug in some rare cases. The same bug can also prevent downloads from working correctly. I won't go into details, but I'm currently preparing a PR to fix it

**Comment by drahti** [comment]
fixed in #19
